### DISCUSSION TITLE
SetParam with QUIC_PARAM_CONN_SEND_BUFFERING

### DISCUSSION THREAD

**Discussion by liwass** [microsoft/msquic#2362] [Q&A] [answered]
QUIC_PARAM_CONN_SEND_BUFFERING seems not supported, at least the define is not found in sources.

Is using 
    Settings.SendBufferingEnabled = FALSE;
    Settings.IsSet.SendBufferingEnabled = TRUE;
achieve the same?

**Comment by nibanks** [comment, marked as answer]
Yes, you just configure it via the settings.
